README.md

```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
-| storage.bucket                             | -                  | Blob storage bucket
-| storage.overrides.*                        | -                  | Key-value pairs to override storage settings
-| storage.createBucket                       | false              | Indicates whether bucket should be created on start-up
-| storage.prefix                             | -                  | Base prefix for all stored resources. Must not contain path separators or any invalid chars
-| encryption.password                        | -                  | Password used for AES encryption
-| encryption.salt                            | -                  | Salt used for AES encryption
-| resources.maxSize                          | 1048576            | Max allowed size in bytes for a resource
-| resources.syncPeriod                       | 60000              | Period in milliseconds, how frequently check for resources to sync
-| resources.syncDelay                        | 120000             | Delay in milliseconds for a resource to be written back in object storage after last modification
-| resources.syncBatch                        | 4096               | How many resources to sync in one go
-| resources.cacheExpiration                  | 300000             | Expiration in milliseconds for synced resources in Redis
-| resources.compressionMinSize               | 256                | Compress a resource with gzip if its size in bytes more or equal to this value
-| redis.singleServerConfig.address           | -                  | Redis single server addresses, e.g. "redis://host:port"
-| redis.clusterServersConfig.nodeAddresses   | -                  | Json array with Redis cluster server addresses, e.g. ["redis://host1:port1","redis://host2:port2"]
-| invitations.ttlInSeconds                   | 259200             | Invitation time to live in seconds
-| access.admin.rules                         | -                  | Matches claims from identity providers with the rules to figure out whether a user is allowed to perform admin actions, like deleting any resource or approving a publication. Example: [{"source": "roles", "function": "EQUAL", "targets": ["admin"]}]. If roles contain "admin, the actions are allowed.
+| Setting                                    | Default            |Required| Description
+|--------------------------------------------|--------------------|--------|-----------------------------------------------------------------------------------------------------------
+| config.files                               | aidial.config.json | No|Config files with parts of the whole config.
+| config.reload                              | 60000              | No|Config reload interval in milliseconds.
+| identityProviders                          | -                  | No|Map of identity providers. **Note**. At least one identity provider must be provided.
```

Comment:
Add all supported IDPs to the example.

Reply:
If we say that at least one provider must be provided - then is it a REQUIRED field?

Reply:
Provide at least one full example with all available parameters

Reply:
What about Okta? @astsiapanay

Reply:
added